packages/nuxt-locokit/src/pages/admin/users/CreateUser/CreateUser.vue

```diff
@@ -164,36 +164,41 @@ import PrimeDropdown from 'primevue/dropdown'
 import { FormGeneric } from '@locokit/designsystem'
 import { Field } from 'vee-validate'
 import { ref } from 'vue'
-import { storeToRefs } from 'pinia'
-import { useStoreUsers } from '../../../../stores/users'
 import { PROFILE } from '../../../../interfaces/toMigrate'
 import { ROUTES_NAMES } from '../../../../paths'
+import { createUser } from '../../../../services/user'
```

Comment:
I think you can use [alias](https://nuxt.com/docs/api/configuration/nuxt-config#alias) provided by nuxt.

Probably like this

```suggestion
import { PROFILE } from '@/src/interfaces/toMigrate'
import { ROUTES_NAMES } from '@/src/paths'
import { createUser } '@/src/services/user'
```

Reply:
Alias are not available here. I think (with the current configuration) that we can only use them on playground.